service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/handlers/NoticeOfChangeEventHandlerEmailTemplateTest.java

```diff
@@ -23,30 +34,42 @@
     private static final String SOLICITOR_FIRST_NAME = "John";
     private static final String SOLICITOR_LAST_NAME = "Watson";
     private static final String EXPECTED_SALUTATION = "Dear John Watson";
+    private static final String CHILD_LAST_NAME = "Jones";
 
     private static final CaseData CASE_DATA = CaseData.builder()
         .id(CASE_ID)
         .caseName(CASE_NAME)
+        .children1(wrapElements(Child.builder()
+            .party(ChildParty.builder()
+                .dateOfBirth(LocalDate.of(2015, 1, 1))
+                .lastName(CHILD_LAST_NAME)
+                .build()).build()))
         .build();
 
+    @MockBean
+    private FeatureToggleService featureToggleService;
```

Comment:
toggle doesn't appear to be used, again it can be defaulted to true

Reply:
moved it to @MockBeans at class level